sdk/automanage/Azure.ResourceManager.Automanage/src/autorest.md

```diff
@@ -0,0 +1,176 @@
+# Generated code configuration
+Run `dotnet build /t:GenerateCode` to generate code.
+``` yaml
+save-inputs: true
```

Comment:
remove this line so the CodeModel files go away

Reply:
@m-nash do you mean line 4?

Reply:
Done